README.md

```diff
@@ -6,5 +6,11 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a library of ruby code that is used as a frameowrk for building websites.
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
 1. Describe the flow of a PATCH request as it goes through the app.
+
+A PATCH request is used to update some part of a resource, not the whole thing.  I am not sure how it goes through the app.
```

Comment:
That is correct. How might you trace the request through your diagram? Can you work out what happens at each step?